analysis/org.eclipse.tracecompass.analysis.chromium.core/src/org/eclipse/tracecompass/internal/analysis/chromium/core/callstack/ChromiumCallStackProvider.java

```diff
@@ -92,20 +92,30 @@ protected boolean considerEvent(@NonNull ITmfEvent event) {
 
     @Override
     protected @Nullable ITmfStateValue functionEntry(@NonNull ITmfEvent event) {
-        if (event instanceof TraceEventEvent && 'B' == (((TraceEventEvent) event).getField().getPhase())) {
+        if (event instanceof TraceEventEvent && isEntry(event)) {
             return TmfStateValue.newValueString(event.getName());
         }
         return null;
     }
 
+    private static boolean isEntry(ITmfEvent event) {
+        char phase = ((TraceEventEvent) event).getField().getPhase();
```

Comment:
Of course, layouts are for later, as soon as I get my hands on this code. It's professional deformation. My JOB is to take a prototype and see how this can be generalized and make to work seemlessly with a maximum of use cases ;-)